examples/native-fungible/index.html

```diff
@@ -171,8 +171,12 @@ <h2>Chain history for <code id="chain-id" class="hex">requesting a new microchai
 
           let errors = [];
           try {
-              const match = recipient.match(/^(0x[0-9a-f]{40})@([0-9a-f]{64})$/);
+              // I failed to make it work with a single regex,
+              // the regex with group alternatives for the owner produces different `match` array
+              // with different group indices so we can't extract it deterministicly when parsing as `recipient` down below.
+              const match = recipient.match(/^(0x[0-9a-f]{40})@([0-9a-f]{64})$/) || recipient.match(/^(0x[0-9a-f]{64})@([0-9a-f]{64})$/);
```

Comment:
I think the way to do it with one regex is

```suggestion
              const match = recipient.match(/^(0x[0-9a-f]{40}|[0-9a-f]{64})@([0-9a-f]{64})$/);
```

No big deal though.

Reply:
I suspect you put the capture group inside the alternative — I tested this one and it works okay.

Reply:
Ah, that might be - yes. I'll redo.